plugins/connection/lxd.py

```diff
@@ -16,7 +16,7 @@
     options:
       remote_addr:
         description:
-            - Container identifier.
+            - Instance (container/VM) identifier.
```

Comment:
I would mention the new feature here:
```suggestion
            - Instance (container/VM) identifier.
            - Since community.general 8.0.0, a FQDN can be provided; in that case, the first component (the part before C(.)) is used as the instance identifier.
```